tasks/publish.yaml

```diff
@@ -56,6 +64,7 @@ tasks:
         default: ${BUNDLE_NAME}
     actions:
       - task: utils:determine-repo
+        if: ${{ eq .variables.TARGET_REPO "" }}
```

Comment:
I put this in here to bypass the utils:determine-repo completely. The idea was if you need to completely set the TARGET_REPO to something else from the specific project, you can set it and what you set will be used instead. I put this in while we were testing publish by posting to GitLab's registry rather than put more github vs gitlab specific stuff inside determine-repo because the inputs themselves are quite a bit different.

You can see this used in `templates/publish.yml` if the person wiring up the job has a specific target repo they want to use. If we are just going to GHCR for everything right now I can take this (and the logic in `templates\publish.yml`) out completely as it won't really be used yet.

Thoughts on that?

Reply:
Thought about it and left the `TARGET_REPO` override in here, but moved the logic that checks it down into `utils:determine-repo` so it's all contained in there. You can still `--set` it on the command line and it works.

Reply:
yep that is more what I meant - will review further!